src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -50,18 +49,7 @@ public String saveItem(Item item) {
         return itemService.fetchAll();
     }
 
-    /*
-    @PostMapping(value ="/item", consumes="application/json", produces = "application/json")
-    @ResponseBody
-    public Item createItem(@ResponseBody Item item){
-        Item newItem = null;
-        try {
-            //newItem = itemService.save(item);
-        } catch (Exception e){
-            //
-        }
-    }
-     */
```

Comment:
Good.

Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.